opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx

```diff
@@ -42,6 +42,8 @@ const useStyles = makeStyles(() => createStyles({
   }),
 }));
 
+export const DEFAULT_DASHBOARD = 'cf093b57-713f-404b-a210-a1c5c8cb3791';
```

Comment:
what is this id ?

Reply:
In order to be able to select the platform default dashboard within UI I added a hardcoded id representing this dashboard.

Reply:
so this "DEFAULT_DASHBOARD" is the default of opencti ? but in the UI we named "default dashboard" the custom one selected for the group, and "platform dashboard" is the default. I think it's completely confusing and will be hard to remember that "default dashboard" in the ui doesn't mean "default_dashboard" in the code.

Reply:
I renamed the constant